studiopress/seo-data-transporter
# Remove Grunt and add pot generation using wp-cli

**Summary of change:**
This PR moves away from Grunt and any npm dependency. It also makes the `.pot` generation available using `wp-cli` and composer.

**This PR has been:**

**Have the changes in this PR been added to the documentation for this project?**
No

**How to test:**
- Download this branch;
- Install the composer dependencies by running `composer install`;
- run `composer makepot` and check if the `.pot` file has been properly generated.
